configure_reclient.py

```diff
@@ -278,6 +287,31 @@ def generate_rewrapper_cfg(self, tool, host_os):
             f'{Paths.reclient_cfgs_dir}/{tool}/rewrapper_{host_os}.cfg',
             rewrapper_cfg, source_cfg_paths)
 
+    def generate_rewrapper_large_cfg(self, tool, host_os):
+        # Load Chromium config for linux remote.
+        rewrapper_cfg_fname = f'linux/{tool}/rewrapper_linux_large.cfg'
+        rewrapper_cfg_file = f'{Paths.reclient_cfgs_dir}/{rewrapper_cfg_fname}'
+        if not os.path.isfile(rewrapper_cfg_file):
+            rewrapper_cfg_file = f'{Paths.script_dir}/{rewrapper_cfg_fname}'
+        rewrapper_cfg = ReclientCfg.parse_from_file(rewrapper_cfg_file)
+
+        # Merge with our configs.
+        source_cfg_paths = [
+            f'{Paths.script_dir}/{tool}/rewrapper_base_large.cfg',
+            f'{Paths.script_dir}/{tool}/rewrapper_{host_os}_large.cfg',
+        ]
+        if self.args.large_pool_name:
+            rewrapper_cfg['platform']['Pool'] = self.args.large_pool_name
+        for source_cfg_path in source_cfg_paths:
+            rewrapper_cfg = ReclientCfg.merge_cfg(rewrapper_cfg,
+                                                  source_cfg_path)
+
```

Comment:
this change is missing the custom merge step. It's important to have it for all configs.
```
        # Launch a custom merge step if it exists.
        if self.custom_py and 'merge_rewrapper_large_cfg' in self.custom_py:
            rewrapper_cfg = self.custom_py['merge_rewrapper_large_cfg'](rewrapper_cfg,
                                                                  tool, host_os)
            source_cfg_paths.append(Paths.custom_py)
```

Reply:
Good point, thank you! https://github.com/EngFlow/reclient-configs/pull/10